app/config/config.test.json

```diff
@@ -32,6 +32,7 @@
 	"membership-application-timeframe-limit": "PT10M",
 	"operator-email": "email@operatorsownmailserver.com",
 	"operator-displayname": "Friendly Operator",
+	"operator-displayname-alternative": "Alternative Operator",
```

Comment:
Wow, sorry. The method `FunFunFactory::setNullMessenger()` wasn't setting the alternative one. All the membership related tests were sending e-mails to your account, because it's in `ValidMembershipApplication`.